packages/foundation/config/tests.js

```diff
@@ -0,0 +1,17 @@
+/** Jest test setup file. */
+
+const { configure } = require('enzyme');
+const Adapter = require('enzyme-adapter-react-16');
+
+// Suppress icon warnings.
+setIconOptions({
+  disableWarnings: true
+});
+
+// Mock requestAnimationFrame for React 16+.
+global.requestAnimationFrame = callback => {
```

Comment:
Do we need this? As far as i know Jest 23 has rAF available

Reply:
I copied a recently made package, so this is templated to an extent. This package currently has no tests so this file will probably change.

Reply:
Looks like I have some cleanup to do ;)